blocksuite/framework/std/src/gfx/interactivity/extension/base.ts

```
@@ -118,6 +119,10 @@ type ActionContextMap = {
       | undefined
     >;
   };
+  elementSelect: {
+    context: ExtensionElementSelectContext;
+    returnType: void;
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)